content/en/synthetics/browser_tests/actions.md

```diff
@@ -234,25 +277,26 @@ To obfuscate local variable values in test results, select **Hide and obfuscate
 
 #### Element
 
-Create a variable from content like a `span` or `div` by extracting the text of this element.
+Create a variable from content such as a `span` or `div` by extracting the text of the element.
 
-#### JavaScript
 
-Write the custom JavaScript code returning the value you want your variable to be assigned to.
+#### JavaScript
 
-**Note**: JavaScript steps support both synchronous and asynchronous code.
+Write the custom JavaScript code that returns the value you want your variable to be assigned to. JavaScript steps support both synchronous and asynchronous code. Because browser tests load external JavaScript by adding the script to the page, they only work if your website accepts external JavaScript.
```

Comment:
Can you clarify what "Write the custom JavaScript code that returns the value you want your variable to be assigned to." means? I am not sure how you would write the code with the response that you want. Seems like it needs context or clarification

Reply:
```suggestion
JavaScript steps support both synchronous and asynchronous code. Because browser tests load external JavaScript by adding the script to the page, they only work if your website accepts external JavaScript.
```